packages/lit-element/package.json

```diff
@@ -9,6 +9,16 @@
   "main": "lit-element.js",
   "module": "lit-element.js",
   "typings": "lit-element.d.ts",
+  "exports": {
+    ".": {
+      "lit-development": "./development/lit-element.js",
```

Comment:
I think we might want to have this condition be just `development` so that the ecosystem can converge on that and dev servers, build tools, and such can have an easy to use `--dev` flag.

Node docs give an example of the `development` condition: https://nodejs.org/api/esm.html#esm_conditional_exports

@LarsDenBakker I'm curious what you think there?

Reply:
Switched to `development`